csharp/src/Apache.Arrow/Apache.Arrow.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.3;netstandard2.0;netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks>netstandard1.3;netstandard2.0;netcoreapp3.1;net6.0</TargetFrameworks>
```

Comment:
How about I remove .NET Standard 1.3 and .NET Core 3.1?

Reply:
The current guidance is to avoid targeting netstandard1.x: https://learn.microsoft.com/en-us/dotnet/standard/library-guidance/cross-platform-targeting

But let's not do it in this PR. Instead, I think we should open a new issue for it. We may want to wait on netcoreapp3.1 though. It was just EOL'd a couple months ago. It sometimes takes people a bit longer to move off these EOL frameworks. It doesn't cost much to keep supporting it for a few months longer.